src/prefect_server/api/flows.py

```diff
@@ -548,6 +549,7 @@ async def schedule_flow_runs(flow_id: str, max_runs: int = None) -> List[str]:
             flow_id=flow_id,
             scheduled_start_time=event.start_time,
             parameters=event.parameter_defaults,
+            labels=event.labels or None,
```

Comment:
I'll be making a Core PR that will make this line irrelevant and we can use `event.labels` all the time